### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by renancouto**
In this PR I replaced `scss-lint` for `stylelint` and add a basic configuration file based on our current standards.

### CODE REVIEW

File: linter/.stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "block-closing-brace-newline-after": "always",
+        "block-opening-brace-space-before": "always",
+        "declaration-colon-space-after": "always",
+        "declaration-colon-space-before": "never",
+        "declaration-property-value-blacklist": {
+            "/^border/": ["none"]
+        },
+        "indentation": 4,
+        "length-zero-no-unit": true,
+        "max-line-length": 120,
```
Comment (janjaap) [inline linter/.stylelintrc.json]: Is this implied or a hard limit?

File: linter/README.md [hunk 2]
```diff
@@ -0,0 +1,368 @@
+# Linter
+
+## Weapon of choice
+We choose [stylelint](http://stylelint.io/) as our Linter because it is the most complete, flexible and it runs on top of Node, that's already a dependency in our workflow.
+
+## Getting started
+Please follow to [official guide](http://stylelint.io/#getting-started) to start using stylelint.
+Still there is no plugin available for JetBrains editors, but [this comment](https://github.com/stylelint/stylelint/issues/346#issuecomment-173925962) might help setting up.
```
Comment (janjaap) [inline linter/README.md]: **There is** no plugin available...

File: linter/README.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+    top: 0;
+}
+
+.bad {
+    top: 0px;
+}
+```
+
+### Max line length
+Less or equal to `120` characters
```
Comment (janjaap) [inline linter/README.md]: What about base64 encoded `url()` values?

File: linter/.stylelintrc.json (hunk 1 above)
Comment (renancouto) [inline linter/.stylelintrc.json]: Right now, all rules are throwing errors, but we can reduce the severity level to **warning** if we consider it is necessary.

File: linter/README.md (hunk 2 above)
Comment (renancouto) [inline linter/README.md]: Atom, Sublime Text an VSCode have. Looks like they're waiting for some API change on JetBrains' editors to write one. Anyways, I going to try to set up the suggested way to see how it works.

File: linter/README.md (hunk 3 above)
Comment (renancouto) [inline linter/README.md]: We can do a few different things here:

- disable only this line (you have to give a reason for disabling a rule)
- reduce the severity of this rule to warning
- create our own plugin that should keep the max-length, but allow base64 urls to be larger than that

Comment (janjaap) [inline linter/README.md]: @renancouto I'd opt for a reduction in severity

**Comment by janjaap** [comment]
@renancouto Not sure if the linter catches it by default, but is the 'each declaration should end with a semi-colon'-rule applied?

**Comment by renancouto** [comment]
@janjaap good catch, it wasn't. Now it is 😉

**Pull request merged**